src/compas_timber/ghpython/components_cpython/CT_Model/code.py

```diff
@@ -58,9 +57,6 @@ def RunScript(
         ##### Adding elements #####
         self.add_elements_to_model(Model, Elements, Containers)
```

Comment:
```suggestion
        self.add_elements_to_model(Model, Elements)
```

I suppose since `containers` input is gone, this should be updated as well.